sample_shavar_list_creation.ini

```diff
@@ -50,6 +50,36 @@ output=analytics-track-digest256
 categories=Social|Disconnect
 output=social-track-digest256
 
+[social-tracking-protection]
+disconnect_url=https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/master/social-tracking-protection-blacklist.json
+categories=Facebook,Twitter,LinkedIn,YouTube
```

Comment:
This syntax is not correct. This does an intersection of those categories (i.e., and AND). You should use the `|` delimiter for OR. So `categories=Facebook|Twitter|LinkedIn|YouTube`.